backburner/templates/hpa.yaml

```diff
@@ -23,21 +29,5 @@ spec:
         averageValue: {{ .value }}
         {{- end }}
   {{- end }}
-  {{- else }}
-  minReplicas: 1
-  maxReplicas: 1
-  metrics:
-  - type: Resource
-    resource:
-      name: cpu
-      target:
-        type: Utilization
-        averageUtilization: 70
-  {{- end }}
-  scaleTargetRef:
-    apiVersion: apps/v1
-    kind: Deployment
-    name: {{ $.Values.name }}-{{ $resourceName }}
----
```

Comment:
don't remove it `---`, because we need to create multiple hpa

Reply:
Fixed, thx!